src/frontend/efiling-frontend/src/components/page/rush/Rush.test.js

```diff
@@ -24,6 +112,17 @@ describe("Rush Component", () => {
     submissionFee,
   };
 
+  const radio1Label =
+    "The attached application is made under Rule 8-5 (1) SCR.";
+  const radio2Label =
+    "The court directed that the order be processed on an urgent basis.";
```

Comment:
is this a line length constraint by a linter? 117-118

Reply:
Yeah I think that's exactly what this is.